modules/generator/processor/spanmetrics/spanmetrics.go

```diff
@@ -58,14 +58,18 @@ func New(cfg Config, registry registry.Registry) gen.Processor {
 		labels = append(labels, sanitizeLabelNameWithCollisions(d))
 	}
 
-	return &Processor{
-		Cfg:                        cfg,
-		registry:                   registry,
-		spanMetricsCallsTotal:      registry.NewCounter(metricCallsTotal, labels),
-		spanMetricsDurationSeconds: registry.NewHistogram(metricDurationSeconds, labels, cfg.HistogramBuckets),
-		spanMetricsSizeTotal:       registry.NewCounter(metricSizeTotal, labels),
-		now:                        time.Now,
+	p := &Processor{}
+	if cfg.Subprocessors["Latency"] {
```

Comment:
Can you move `"Latency"` and `"Count"` to constants? Even maybe using the `SubProcessor` type suggested by @zalegrala

Reply:
@mapno I will push up something that has the Subprocessor type